package.json

```diff
@@ -13,6 +13,7 @@
     "flow-typed": "rm -rf ./flow-typed && flow-typed install",
     "karma": "cross-env NODE_ENV=test babel-node --plugins=transform-es2015-modules-commonjs ./node_modules/.bin/karma start",
     "lint": "eslint src/ server/ test/ *.js",
+    "lint:fix": "eslint --fix src/ server/ test/ *.js",
```

Comment:
I've seen that this can be dangerous.  Can we remove this?

Reply:
We are caught up on linting for this repo so no reason we can't fix individually.

Reply:
This is fine, you have to run this manually. 👍 same as `npm run lint -- --fix`

Reply:
(FWIW can also set up vscode to auto-fix on save, super useful)

Reply:
And it is! I can, however my default configuration seems not to be the same required here, so my auto formatting is not suitable. I use `esbenp.prettier-vscode` ... any suggestion about how to configure it properly??

I'll remove that line btw